assets/js/googlesitekit/api/cache.js

```diff
@@ -22,12 +22,134 @@
 import { isEqual } from 'lodash';
 
 /**
- * Internal dependencies
+ * Prefix used for all Site Kit keys.
+ *
+ * Anything not using this key should not be touched by this library.
+ *
+ * @private
+ */
+export const STORAGE_KEY_PREFIX = 'googlesitekit_';
+
+const defaultOrder = [ 'localStorage', 'sessionStorage' ];
+let storageBackend;
+let storageOrder = [ ...defaultOrder	];
+/**
+ * Override the storage backend.
+ *
+ * Largely used for tests. Should not be used directly.
+ *
+ * @private
```

Comment:
There should always be `since` annotations (right above `private`). If it's for the upcoming version, use `since n.e.x.t`.

Reply:
Ah, right. We may need to lint for that; it's easy to forget.

I've added it to any JSR function missing it; it wasn't in much of the original API and cache code I wrote.